backend/app/routers/reservations.py

```
@@ -0,0 +1,298 @@
+    return [reservation_to_list_dict(r) for r in rows]
+
+
+# ---------------------------------------------------------------------------
+# Public: visitor self-lookup by e-mail
+# ---------------------------------------------------------------------------
+
+
+@router.get("/my", response_model=list[ReservationGuestOut])
+async def my_reservations(
+    email: Annotated[EmailStr, Query(description="E-mail address used when making the reservation")],
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)